_config/config.yml

```diff
@@ -4,7 +4,7 @@ Name: 'jsontextconfig'
 
 Injector:
   JSONText:
-    class: \phptek\JSONText\Fields\JSONText
+    class: \phptek\JSONText\ORM\FieldType\JSONText
     backend: postgres
```

Comment:
Strictly speaking we don't need these prepended backslashes anyway in YML config.